src/VisualStudio/Core/Def/Implementation/EditAndContinue/VisualStudioActiveStatementProvider.cs

```diff
@@ -78,10 +72,19 @@ void CancelWork()
                                     return;
                                 }
 
-                                if (activeStatementsResult.ErrorCode != 0)
+                                switch (activeStatementsResult.ErrorCode)
                                 {
-                                    unexpectedError = activeStatementsResult.ErrorCode;
-                                    return;
+                                    case 0:
```

Comment:
nit, VSConstants.S_OK

Reply:
We do not have this switch/case anymore.

The current code is != or == 0 which confirms the original code. Let us keep it.


---
In reply to: [198272677](https://github.com/dotnet/roslyn/pull/28070#discussion_r198272677) [](ancestors = 198272677)